scripts/lint.py

```diff
@@ -47,8 +49,10 @@ def main(args):
   os.chdir(settings.GetRoot())
   try:
     cl = git_cl.Changelist(auth_config=auth_config)
-    change = cl.GetChange(cl.GetCommonAncestorWithUpstream(), None)
-    files = [f.LocalPath() for f in change.AffectedFiles()]
+    upstream = git_common.get_or_create_merge_base(cl.GetBranch(), options.base_branch)
+    changed_files_cmd = git_cl.BuildGitDiffCmd('--name-only', upstream, "")
```

Comment:
I'm still confused by this. Why aren't we just calling `cl.GetChange` with a different branch name?

Reply:
`git_common.get_or_create_merge_base(self.GetBranch(), upstream)` passed instead of `cl.GetCommonAncestorWithUpstream()`

Reply:
@bridiver - Using cl.GetChange with upstream invokes this error: 

```
python /Users/pranjal/brave/60x/brave-browser/scripts/lint.py --project_root=/Users/pranjal/brave/60x/brave-browser/src --base_branch=origin/master
ERROR: Your diff contains 2 commits already in refs/remotes/origin/master_lint_dummy.
Run "git log --oneline 1394f548994edb65d179147f5a7ee97e359586db..HEAD" to get a list of commits in the diff.  If you are using a custom git flow, you can override the reference used for this check with "git config gitcl.remotebranch <git-ref>"
```

I'm not sure if there is a way around this.